# 捕获异常

执行下面的代码，捕获触发的异常。

```python
def func():
    raise Exception('Hello World!')


try:
    func()
except Exception as e:
    print(e)
print('Hello Python!')
```

